### DISCUSSION TITLE
Watercooler 🐳

### DISCUSSION THREAD

**Discussion by Buzzpy** [Buzzpy/Dev-Encyclopedia#80] [General]
Feel free to chat about anything here: from blaming the maintainers to celebrating new features. 🍻

![GIF Minions](https://github.com/user-attachments/assets/15a74a81-4ccb-4c8d-87b7-8158f5bfb60b)

**Comment by Sudharshaun** [comment]
Hey @Buzzpy I think you need to sync master and develop branches. Develop is behind master.

**Reply by Buzzpy** [reply]
Done and dusted! 🧹

**Comment by RayMathew** [comment]
@Buzzpy may I ask what's the average view count of the website, now that it's been up for a while? Do you have any specific ideas for the future roadmap?

**Reply by Buzzpy** [reply]
Hi!
Sorry for not getting back to you sooner, I was busy all week. The average view count has decreased compared to the initial launch, but it's around 1K, not exceeding 2K. 

Regarding the future, I don't have any specific, new ideas, except for the website's pagination. What about you, by the way?

**Reply by RayMathew** [reply]
Oh ok! Am glad to know that it's reached a stable view count. People are able to see the value we hoped they would see :)

I did have a few ideas.. just no time to work on them, because I'm building some of my own projects.

- Filter by keywords / topics in each modal (I had mentioned this in the comments of the Issue I had worked on).
- Pressing '/' gives focus to search box. This would bring uniformity with Google and YouTube.
- Dark mode the color scheme could be improved.
- Dark mode I was thinking of moving it to the top right corner as a tiny toggle, since it's not actually part of the core mechanics of the website.
- Improve the search by making it Fuzzy Search. Let it be tolerant of spelling mistakes when users search.
- Improve the accessibility score of the website. (Haven't checked what it is right now).

**Reply by Buzzpy** [reply]
Awesome ideas! I should start working on what I can too, soon.
Agreed on improving the scores, I think I should focus on best practices too. Accessibility on 95 on both mobile and desktop, a good score I guess!

**Reply by KC900201** [reply]
Sorry to interrupt here. Regarding the accessibility score, can I suggest that we can also invest some time on researching how to improve the SEO (search engine optimization) score for our project.

I have some ideas as well for the future roadmap:

- Improve the responsiveness of the web layout by implementing UI libraries for the web revamp
- Migrate the existing terms glossary to Astro framework [database tool](https://docs.astro.build/en/guides/astro-db/)
- Implementing proofreading and spell check for every new term introduced in the web glossary (mentioned in issue #173)

**Comment by KC900201** [comment]
@Buzzpy may I know if there's any plan to improve the view count of the webpage by tuning the SEO?
